drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api_common.c

```diff
@@ -24,6 +24,25 @@
 #include "fmac_bb.h"
 #include "util.h"
 
+#define FT_PROG_VER1 1
+#define FT_PROG_VER2 2
+#define FT_PROG_VER3 3
+#define FT_PROG_VER1_2G_DSSS_TXCEIL_BACKOFF 0
+#define FT_PROG_VER1_2G_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER1_5G_LOW_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER1_5G_MID_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER1_5G_HIGH_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER2_2G_DSSS_TXCEIL_BACKOFF 0
+#define FT_PROG_VER2_2G_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER2_5G_LOW_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER2_5G_MID_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER2_5G_HIGH_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER3_2G_DSSS_TXCEIL_BACKOFF 0
+#define FT_PROG_VER3_2G_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER3_5G_LOW_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER3_5G_MID_OFDM_TXCEIL_BACKOFF 0
+#define FT_PROG_VER3_5G_HIGH_OFDM_TXCEIL_BACKOFF 0
```

Comment:
Move these to phy_rf_params.h or rpu_if.h (as appropriate).

Reply:
Moved all the macros related to back off values are moved to phy_rf_params.h file.